README.md

```
@@ -0,0 +1,154 @@
+        <img src="https://github.com/Airlock-Auth/airlock-auth/assets/47309835/aa360624-19b2-427a-839f-3e3e004181d1" />
+    </picture>
+</p>
+
+
+## Airlock
+
+Airlock is a Next JS project by Rory Garton-Smith. Airlock provides a method of securing yourself against 2FA SMS security limitations.
+
+This repo allows you to build your own version of Airlock and self-host.
+
+#### Security Warning:
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)